src/lib/change-email-helpers.js

```diff
@@ -0,0 +1,118 @@
+const helpers = require('../lib/helpers');
+const uuid = require('uuid/v4');
+const moment = require('moment');
+const { pool } = require('../lib/connectors/db');
+const DB = require('./connectors/db');
+const { EmailChangeError } = require('../controllers/change-email');
+const Repository = require('@envage/hapi-pg-rest-api/src/repository');
+
+class UsersRespository extends Repository {
+  findById (userId) {
+    const user = this.find({ user_id: userId });
+    return user.data[0];
+  }
+  checkEmailAddress (verificationId, newEmail) {
+    const query = `SELECT user_name FROM idm.users u
+      JOIN (
+        SELECT application
+        FROM idm.users u
+        JOIN idm.change_email_verification v ON v.user_id = u.user_id
+        WHERE verification_id=$1
+      ) v ON v.application=u.application
+      WHERE u.user_name=$2`;
+
+    return DB.query(query, [verificationId, newEmail]);
```

Comment:
The class this one inherits from already has a DB query call so we could do this.dbQuery here:
https://github.com/DEFRA/hapi-pg-rest-api/blob/master/src/repository.js#L22